code/src/java/pcgen/core/PlayerCharacter.java

```diff
@@ -10085,6 +10057,6 @@ public String getControl(String string)
 
 	public boolean hasControl(String string)
 	{
-		return controller.get(ObjectKey.getKeyFor(String.class, "*" + string)) != null;
+		return controller.containsKey(ObjectKey.getKeyFor(String.class, "*" + string));
```

Comment:
IF the value can be set to null, these are not equivalent.  Have you checked for that case?